src/main/groovy/org/mockito/release/internal/gradle/AutoVersioningPlugin.java

```diff
@@ -28,67 +28,23 @@
  * Adds following tasks:
  *
  * <ul>
- *     <li>gitAddBumpVersion</li>
- *     <li>gitCommitBumpVersion</li>
- *     <li>gitPushBumpVersion</li>
  *     <li>bumpVersionAndPush</li>
  * </ul>
  */
 public class AutoVersioningPlugin implements Plugin<Project> {
 
     static final String BUMP_VERSION_AND_PUSH_TASK = "bumpVersionAndPush";
```

Comment:
Let's start avoiding 'and' in the name of the tasks for clarity and ease of use. I suggest something like that: "pushBumpVersion" or "performBumpVersion". The latter is interesting. We can come up with a naming convention that tasks that start with "perform" are coarse grained, high level tasks.